examples/index.json

```
@@ -378,6 +378,9 @@
         "samples": [{
                 "name": "Error Handling",
                 "url": "error-handling"
+            },{
+                "name": "Error type",
```

Shall we make this the last BBE in the error section? IMO it'll feel more natural to talk about return, check, panic etc. together. We also don't seem to be using user defined errors in the other error examples, so that's not an issue either.

Shall we also rename this as
```suggestion
                "name": "User-defined Errors",
```